index.html

```diff
@@ -22,14 +22,14 @@
                 </div>
             </nav>
         </header>
-        <section class="section2">
+        <section class="hero">
```

Comment:
- [ ] Kindly fix this hero section as mentioned by one of the previous reviewers but your design is still the same as it was before check below :point_down:  and fix this also :slightly_smiling_face: 

You Design:
![image](https://user-images.githubusercontent.com/88894525/219875203-183db0fa-4959-4f36-8d77-a65ed1fdafe7.png)

Figma Design:
![figma](https://user-images.githubusercontent.com/88894525/219874756-1915bfa0-ec3e-42e0-a4c2-4f26b3e56bd4.png)